File: Problems/No.168_Excel_Sheet_Column_Title/excel_sheet_column_title.cpp
class Solution {
public:
    string convertToTitle(int columnNumber) {
        
        // parameter
        string ans = "";
        
        // start compute
        columnNumber--;
        while (columnNumber >= 0) {
            
            ans = char(65+columnNumber%26) + ans;
            columnNumber /= 26;
            columnNumber--;
            
        }
        
        return ans;
        
    }
};

File: Problems/No.168_Excel_Sheet_Column_Title/excel_sheet_column_title.py
class Solution(object):
    def convertToTitle(self, columnNumber):
        """
        :type columnNumber: int
        :rtype: str
        """
        
        # parameters
        mapping = ['A', 'B', 'C', 'D', 'E',
                   'F', 'G', 'H', 'I', 'J',
                   'K', 'L', 'M', 'N', 'O',
                   'P', 'Q', 'R', 'S', 'T',
                   'U', 'V', 'W', 'X', 'Y', 'Z']
        ans = ""
        
        # start compute
        columnNumber -= 1
        while (columnNumber >= 0):
            ans = mapping[columnNumber%26] + ans;
            columnNumber //= 26
            columnNumber -= 1
        
        return ans
